### PR TITLE
[INTEL MKL] Added cpu docker file with python36.

### PR DESCRIPTION
**Pull request by rsketine**

### CODE REVIEW

**Comment by rsketine** [comment]
@penpornk can you please review this. Out public PR testing is running with python3.5 which is causing many failures would like to move to py36. Thank you,

**Comment by penpornk** [comment]
@rsketine Sorry for the delay! Since this is more about the generic TensorFlow build, I think @av8ramit is a better reviewer.

File: tensorflow/tools/ci_build/Dockerfile.cpu-py36 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Jan Prach <jendap@google.com>"
```
Comment (av8ramit) [inline tensorflow/tools/ci_build/Dockerfile.cpu-py36:3]: Please update this with your email.

**Comment by rsketine** [comment]
@av8ramit  can you please review this so that our public CI can start using python36.

**Review by av8ramit — APPROVED** [review]

**Pull request merged**